docs(s3): update lifecycle rule & MPU documentation MTA-5049 (#3697)
scaleway/docs-content: storage/object/how-to/manage-lifecycle-rules.mdx

6. Click **Disable lifecycle rule** to confirm.

<Message type="note">

> Once disabled, the lifecycle rule remains listed and can be enabled again. Click <Icon name="more" />, then select **Activate** to do so.

Reviewer: ```suggestion
Once disabled, the lifecycle rule remains listed and can be enabled again. Click <Icon name="more" />, then select **Activate** to re-enable it.
```
